spacialgaze-plugins/leaderhelp.js

```diff
@@ -53,8 +53,8 @@ exports.commands = {
 			'<b>/takecurrency [user], [amount]</b> - removes a special amount of Stardust from a user (needs a reason) <br />' +
 			'<b>/moneylog [number]</b> - to view the last x lines <br />' +
 			'<b>/moneylog [text]</b> - to search for text <br />' +
-			'<b>/pmall [message]</b> - sends a pm to all users connected to SG <br />' +
-			'<b>/pmallstaff [message] or /staffpm [message]</b> - sends a pm to all staff members connected to SG <br />' +
+			'<b>/pmall [message]</b> - sends a pm to all users connected to WL <br />' +
```

Comment:
dont abbreviate hwere it isnt needed, since WL dosent make as much sense as SG as an abbreviation